eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -468,4 +468,13 @@
     </Or>
     <Bug pattern="PZLA_PREFER_ZERO_LENGTH_ARRAYS"/>
   </Match>
+
+  <!-- It is fine to have un-used variables, unread fields, anonymous static inner classes in javadoc code samples. -->
```

Comment:
@hemantkr @JonathanGiles I hope it's OK that I added these globbed spotbug excludes for code samples. I am not sure if it was on your plate to look at Hemant.

Reply:
Yes, this is what I asked for in a different PR. I'm not sure what ever happened to that request...